packages/terra-form/tests/nightwatch/textarea/textarea-spec.js

```diff
@@ -2,15 +2,61 @@
 const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
 
 module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
+  'Sets the rows attribute to the value of the passed in rows prop': (browser) => {
+    browser
+      .url(`${browser.launchUrl}/#/tests/form-tests/textarea/rows`)
+      .assert
+      .attributeEquals('textarea', 'rows', '15');
+  },
   'Displays a default Textarea with a number input': (browser) => {
     browser
       .url(`${browser.launchUrl}/#/tests/form-tests/textarea/default`)
-      .assert.elementPresent('textarea');
+      .assert
+      .elementPresent('textarea');
   },
-
   'Displays a populated Textarea with correct name': (browser) => {
     browser
       .url(`${browser.launchUrl}/#/tests/form-tests/textarea/populated`)
-      .assert.elementPresent('textarea[name="job_experience"]');
+      .assert
+      .elementPresent('textarea[name="job_experience"]');
+  },
+  'Displays a small textarea with the correct rows': (browser) => {
+    browser
+      .url(`${browser.launchUrl}/#/tests/form-tests/textarea/small`)
+      .assert
```

Comment:
nit: using linebreaks like this is inconsistent with other test formats

Reply:
Updated Here. https://github.com/cerner/terra-core/pull/966/commits/e9f9c0e713cadcc694eb400d9c14098e40a3ae31